### PR TITLE
Home page 

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/components/UI/AddPost/AddPost.jsx [hunk 1]
```diff
@@ -46,7 +30,11 @@ const AddPost = () => {
...
         console.log(postId);
-        Post.userId = userState.user.userId ?? "";
+        Post.user = userState.user ?? "";
+        Post.date = new Date();
+        Post.createdTime = new Date().getTime();
+        Post.createdAt = new Date().toDateString();
+        Post.updatedAt = new Date().toDateString();
         console.log(Post);
         const userRef = doc(firestore, `posts/${postId}`);
         console.log(Post)
```
Comment (SinchanaMS) [inline src/components/UI/AddPost/AddPost.jsx:45]: Remove logs after testing.

File: src/pages/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -1,18 +1,47 @@
+import {useEffect, useState} from 'react';
 import AddPost from 'components/UI/AddPost/AddPost';
-import { NormalButton } from 'components/UI/Buttons/buttons';
-import { IconEmojiInput, IconGifInput, IconImageInput, IconsProfile } from 'components/UI/Icons/Icons';
 import React from 'react';
```
Comment (SinchanaMS) [inline src/pages/HomePage/HomePage.jsx:3]: You can remove this import or add it to line 1.

File: src/utils/postService.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    try {
+        const response = await getDocs(collectionRef);
+        console.log(response.docs)
+        console.log(response.docs.map(i => {
+            return { ...(i.data()[i.id]), postid :i.id }
+        }))
+        // setData(res1.data() ?? {});
+    } catch (err) {
+        console.log(err);
+        // Alert("error", err.meskksage);
```
Comment (SinchanaMS) [inline src/utils/postService.js:35]: Remove unused code and logs.

**Review by SinchanaMS — APPROVED** [review]
The code looks good. 
Found a few logs that you can remove if you're done testing.
The deploy preview isn't working. Please check.

**Review by m26unkwn — APPROVED** [review]
Could not find any issues related to code as it most  of the code is related to firebase.

Otherwise code is quality is good.
And you can just remove the unwanted console.

File: src/components/UI/AddPost/AddPost.jsx (hunk 1 above)
Comment (tarunsankhla) [reply src/components/UI/AddPost/AddPost.jsx:45]: okay sure

File: src/pages/HomePage/HomePage.jsx (hunk 2 above)
Comment (tarunsankhla) [reply src/pages/HomePage/HomePage.jsx:3]: done

File: src/utils/postService.js (hunk 3 above)
Comment (tarunsankhla) [reply src/utils/postService.js:35]: a lot of try and error going i need those logs for now i will remove them once they are integrated and tested

**Pull request merged**